src/declarations/backend/backend.did.d.ts

```diff
@@ -27,6 +27,12 @@ export interface CanisterStatusResultV2 {
 	module_hash: [] | [Uint8Array | number[]];
 }
 export type CanisterStatusType = { stopped: null } | { stopping: null } | { running: null };
+export interface Config {
+	ecdsa_key_name: string;
+	allowed_callers: Array<Principal>;
+	supported_credentials: [] | [Array<SupportedCredential>];
+	ic_root_key_raw: [] | [Uint8Array | number[]];
```

Comment:
just for my info: wouldn't it always be `Uint8Array`? In the sense, is it not redundant with `number[]`? Or we expect that they could be passed as `number` and parsed after?

Reply:
There is no rules, from experience I've got the feeling that I use more often `Uint8Array` in NodeJS context and `number[]` in browser context but probably depends of the features.

As Max said, we have to live with those types, there are automatically generated and defined by Candid.